.github/workflows/shared-wildfly-build-and-test.yml

```diff
@@ -0,0 +1,98 @@
+name: Build and Test WildFly
+
+on:
+  workflow_call:
+    inputs:
+      test-arguments:
+        description: "Test arguments"
+        required: false
+        default: ""
+        type: string
+      build-arguments:
+        description: "Build arguments"
+        required: true
+        default: ""
+        type: string
+      maven-repo-path:
+        description: "Maven repo path to extract to local cache"
+        required: true
+        default: ""
+        type: string
+      maven-repo-name:
+        description: "Maven repo name to extract to local cache"
+        required: true
+        default: ""
+        type: string
+      java-versions:
+        description: "JDK versions array"
+        required: true
+        default: ""
+        type: string
+      os:
+        description: "OS array"
+        required: true
+        default: ""
+        type: string
```

Comment:
We should remove the default here if they are required.

Reply:
Done